examples/xgboost_results.csv

```diff
@@ -0,0 +1,29 @@
+0,tox21,index,classification,train,xgb_classifier,0.89057892184400222,valid,xgb_classifier,0.75334807887366162,time_for_running,525.361398935318

```

Comment:
Also, this result file maybe a bit redundant as well, since we typically keep all results in `README.md` and our jenkins configuration. We could help update and check all the performances, no worries.